src/dashboard/src/settings/components/ldap_auth.py

```diff
@@ -29,9 +29,9 @@
     )
 AUTH_LDAP_USER_DN_TEMPLATE = environ.get("AUTH_LDAP_USER_DN_TEMPLATE", None)
 AUTH_LDAP_USER_ATTR_MAP = {
-    "first_name": "givenName",
-    "last_name": "sn",
-    "email": "mail",
+    "first_name": environ.get("AUTH_LDAP_USER_ATTR_FIRST_NAME", "givenName",),
+    "last_name": environ.get("AUTH_LDAP_USER_ATTR_LAST_NAME", "sn",),
+    "email": environ.get("AUTH_LDAP_USER_ATTR_EMAIL", "mail",),
```

Comment:
Are these trailing commas inside the `get(...)` calls needed?

Reply:
Removed, and renamed to the examples above!